xrayutilities/simpack/models.py

```diff
@@ -913,3 +1352,4 @@ def densityprofile(self, nz, plot=False):
             plt.tight_layout()
 
         return z, prof
+
```

Comment:
this is clearly the largest contribution and I am interested to merge this. 
It would be important for me to have a working example script using the functionality of the new models. Do you have something appropriate?

Reply:
Yes. See attached.
Once added to /examples I can modify.
[dynaR_example.txt](https://github.com/dkriegner/xrayutilities/files/2281076/dynaR_example.txt)

Reply:
I have few questions to the example:

* It seems your code does not use the 'resolution_width' parameter: So I suggest to remove it from the example code, agreed?
* also it seems your code does not use the 'sample_width', 'beam_width' parameter. Again I would remove it for the moment and discuss how to handle that better at a later stage

My main comment, however, is:
In the example you call my SpecularReflectivityModel "kinematical". I strongly disagree to that...
What I see so far is that your code produces exactly the same result as mine :) see below:

I still need to benchmark it to see which one is faster

![image](https://user-images.githubusercontent.com/9802901/44160671-634b2e00-a0bb-11e8-96a3-50b30ab90a86.png)

I understand that your code can calculate 'S' and 'P' separately... I guess this could be also added to mine if needed...